### DISCUSSION TITLE
Any chance of supporting multi-byte characters?  (And therefore emoji?)

### DISCUSSION THREAD

**Discussion by brandonzylstra** [sqids/sqids-ruby#4] [Ideas]
I noticed in the documentation that multi-byte characters are not currently supported.  Is this something that might change?

**Comment by 4kimov** [comment]
Might, but likely [not soon](https://github.com/orgs/sqids/discussions/1). Adds a lot of complexity, but we haven't seen a valid use-case / much demand _yet_. You could help push it forward by sharing your use-case in that discussion.
